alertmanager/multitenant.go

```diff
@@ -21,12 +22,55 @@ import (
 	configs "github.com/weaveworks/cortex/configs/client"
 	"github.com/weaveworks/cortex/util"
 	"github.com/weaveworks/mesh"
+	"strings"
 )
 
 const (
 	// Backoff for loading initial configuration set.
 	minBackoff = 100 * time.Millisecond
 	maxBackoff = 2 * time.Second
+
+	statusPage = `
+<!doctype html>
+<html>
+	<head><title>Cortex Alertmanager Status</title></head>
+	<body>
+		<h1>Cortex Alertmanager Status</h1>
+        <h2>Mesh router</h2>
```

Comment:
Copy/paste from writing the template in an HTML buffer. Go & Make are the only languages where I actually use tab literals.